pom.xml

```diff
@@ -146,6 +146,7 @@
 		<version.checkstyle.plugin>3.1.0</version.checkstyle.plugin>
         <version.properties.plugin>1.0.0</version.properties.plugin>
         <version.dependency.plugin>3.1.1</version.dependency.plugin>
+        <version.assembly.plugin>3.1.1</version.assembly.plugin>
```

Comment:
Didn't we use assembly plugin before?

Reply:
It is in version`2.2-beta-5` and the version is defined out of the scope of the project. I think this should be definel locally.